src/typing/typeloadModule.ml

```diff
@@ -791,7 +791,51 @@ let type_module ctx mpath file ?(dont_check_path=false) ?(is_extern=false) tdecl
 
 let type_module_hook = ref (fun _ _ _ -> None)
 
-let load_module' ctx g m p =
+let rec get_reader ctx input mpath p =
+	let make_module path file =
+		let m = ModuleLevel.make_module ctx path file p in
+		m.m_extra.m_processed <- 1;
```

Comment:
Is this necessary? With the initial `compilation_step = 1` it seems fine to leave this at 0.

Reply:
`is_cached` explicitly checks for `m.m_processed <> 0`

Reply:
Yes but is that necessary if the compilation step starts at 1 anyway?